recipe/meta.yaml

```diff
@@ -25,25 +25,31 @@ requirements:
     - python >=3.6
     # Required
     - affine >=2.2
+    - blas =*.*=mkl
     - click >=7.0
     - cmocean >=2.0
     - dask >=1.2
     - fiona >=1.8
-    - gdal >=2.4,<3.0
-    - matplotlib-base >=3.0
+    - gdal >=2.4
+    - jdcal >=1.4.1
+    - geopandas >=0.6.2
+    - jsonschema >=3.2.0
+    - matplotlib >=3.0
     - netcdf4 >=1.5.0
     - numba >=0.43
     - numpy >=1.16
     - pandas >=0.24
     - pillow >=6.0
     - proj4 >=5.2
+    - pyjwt >=1.7
     - pyproj >=2.1
     - pyyaml >=5.1
     - rasterio >=1.0
     - s3fs >=0.2
     - scipy >=1.2
     - setuptools >=41.0
     - shapely >=1.6
+    - sphinx >=2.0.1
```

Comment:
Is sphinx needed for the conda package?

Reply:
Hmm, maybe this rather belongs in the "host" section...  I will look into it.

Reply:
No, not needed! Get rid of it.

Reply:
Done.